website/docs/r/org_vdc.html.markdown

```diff
@@ -72,7 +72,7 @@ The following arguments are supported:
 * `name` - (Required) VDC name
 * `description` - (Optional) VDC friendly description
 * `provider_vdc_name` - (Required, System Admin) Name of the Provider VDC from which this organization VDC is provisioned.
-* `allocation_model` - (Required) The allocation model used by this VDC; must be one of {AllocationVApp ("Pay as you go"), AllocationPool ("Allocation pool"), ReservationPool ("Reservation pool")}
+* `allocation_model` - (Required) The allocation model used by this VDC; must be one of {AllocationVApp ("Pay as you go"), AllocationPool ("Allocation pool"), ReservationPool ("Reservation pool"), Flex ("Flex")}
```

Comment:
Let's use a system to mark options supported only in newer vCD versions here too, in the same way as we discussed here:
https://github.com/terraform-providers/terraform-provider-vcd/pull/443#discussion_r376924440

```suggestion
* `allocation_model` - (Required) The allocation model used by this VDC; must be one of {AllocationVApp ("Pay as you go"), AllocationPool ("Allocation pool"), ReservationPool ("Reservation pool"), Flex ("Flex") *v2.7+*, *vCD 9.7+*)}
```

Also, this list of models is becoming hard to read. Do the curled braces serve any purpose? Can we make it a bulleted list instead?

```suggestion
* `allocation_model` - (Required) The allocation model used by this VDC. Supported values:
    * AllocationVApp ("Pay as you go")
    * AllocationPool ("Allocation pool")
    * ReservationPool ("Reservation pool")
    * Flex ("Flex") (*v2.7+*, *vCD 9.7+*)
```

CC: @Didainius

Reply:
changed

Reply:
Thanks, the resulting docs look quite nice!

![Screenshot 2020-02-12 at 12 27 38](https://user-images.githubusercontent.com/11330689/74326447-10773300-4d93-11ea-848c-603502459d0c.png)